website/docs/concepts/chaoshub.md

```diff
@@ -4,4 +4,88 @@ title: ChaosHub
 sidebar_label: ChaosHub
 ---
 
-## Coming soon
+---
+
+## Introduction
+ChaosHub allows a user to orchestrate workflows from the Public **[ChaosHub](http://hub.litmuschaos.io/)** or an alternate source for the Experiments (basically, a **[fork](https://github.com/litmuschaos/chaos-charts)** of the public hub with custom experiments).
```

Comment:
We should use the second person to address, as this convention is used everywhere. You can write - `ChaosHub allows a user` -> `ChaosHub allows a you`